Gandalf/build.sh

```diff
@@ -1,119 +1,98 @@
 #!/bin/bash
+# 
+# Packages "Gandalf" into a deb
+#
+# Follow https://google.github.io/styleguide/shell.xml
```

Comment:
Better: Follow <Link> for the styling of this script, so it's better readable and unified.